sunpy/io/special/asdf/extension.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from asdf import AsdfExtension
+from asdf.util import filepath_to_url
+
+from .types import _sunpy_tags
+
+__all__ = ['SunpyExtension']
+
+
+SUNPY_SCHEMA_URI_BASE = 'http://sunpy.org/schemas/'
```

Comment:
Link does not work.

Reply:
it's not supposed to. Well it dosen't have to.